src/main/java/com/jcabi/github/mock/MkCommits.java

```diff
@@ -112,4 +110,9 @@ public Commit get(
     ) {
         return new MkCommit(this.storage, this.self, this.coords, sha);
     }
+
+    @Override
+    public Statuses statuses(@NotNull(message = "sha can't be NULL") String sha) {
+        return null;
```

Comment:
@zygm0nt please, throw ```UnsupportedOperationException```  here instead of returning null and create a todo to implement it later